lib/exception_hunter/error_creator.rb

```diff
@@ -1,14 +1,16 @@
 module ExceptionHunter
   class ErrorCreator
+    TAGS = %w[HTTP Worker Manual].freeze
```

Comment:
There should be one constant per tag and use them. Alternatively, it could be a hash